src/components/LoginPanel.tsx

```diff
@@ -8,16 +8,16 @@ import React, { useEffect } from "react";
 import styled, { useTheme } from "styled-components";
 import Title from "@/components/Title/Title";
 import { NavBarHeight } from "@/components/NavBar/NavigationBar";
+import Paper from "@mui/material/Paper";
 
 export const LoginMain = styled.main`
     height: calc(100vh - ${NavBarHeight});
     display: flex;
     align-content: center;
 `;
 
-const MiddleDiv = styled.div`
+const MiddleDiv = styled(Paper)`
     max-width: 400px;
-    box-shadow: 0 0 15px ${(props) => props.theme.shadow};
     border-radius: 10px;
```

Comment:
Think there could be potential for a green border to look nice? Not sure though, but give it a go maybe.